src/databricks/labs/ucx/install.py

```diff
@@ -806,6 +825,25 @@ def _job_clusters(self, names: set[str]):
             )
         return clusters
 
+    def _cluster_policy_definition(self, conf: dict) -> str:
+        policy_definition = {
+            "spark_version": {"type": "fixed", "value": self._ws.clusters.select_spark_version(latest=True)},
+            "custom_tags.ResourceClass": {"type": "fixed", "value": "SingleNode"},
+            "node_type_id": {"type": "fixed", "value": self._ws.clusters.select_node_type(local_disk=True)}
+        }
+        if conf:
+            for key, value in conf.items():
+                policy_definition[f"spark_conf.{key}"] = {"type": "fixed", "value": value}
+        if self._ws.config.is_aws:
```

Comment:
what if we have AWS Glue? should we copy properties from there? 
what about external metastores?